src/panels/config/blueprint/ha-blueprint-overview.ts

```diff
@@ -155,6 +157,16 @@ class HaBlueprintOverview extends LitElement {
     })
   );
 
+  protected firstUpdated(changedProps: PropertyValues) {
+    super.firstUpdated(changedProps);
+    if (this.route.path === "/import") {
```

Comment:
The downside of this approach is that after adding the blueprint, pressing the back button will open the dialog again. Should we replace the url with the blueprint dashboard?